azure-devops-case-study.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Azure DevOps Case Study
+keywords: development platforms, Azure DevOps, Microsoft
+tags: Development Platforms, Azure DevOps, Microsoft
+sidebar: development-platforms
+permalink: development-platforms/azure-devops.html
+folder: development-platforms
+---
+
+## Introduction
+Azure DevOps is a cloud based platform that is used to manage the life span of software development projects, and the main purpose of Azure DevOps is to provide tools and services that allow the development team to work together on planing, development, testing ect. 
+
+
+## Brief History
+Azure is owned by microsoft and was originally called Visual Studio Team Services (VSTS). VSTS was crated in 2015 and changed its name to Azure DevOps in 2018. the re-branding of VSTS was done like a statement to make it clear that they would focus more on DevOps-tools and cloud services. 
+
+VSTS (Azure DevOps) has become a very popular Dev-platform since its release back in 2015, but after the name change the platforms popularity went to the sky, this may have some thing to do with users seeing the Microsoft was creating a serious and professional Developing platform with a lot of awesome and convenient tools to use. 
+
+today Azure is used by a lot of big companies and continues to grow.
+
+## Features
+
+    * Azure Boards (planing and management tool)
+    * Azure Repos (source code control tool)
+    * Azure Test Plans (test automation & reporting)
+    * Azure Artifacts (package handling)
+    * Azure Pipelines (helps automate build, testing and distribution processes)
+
+    Azure DevOps also gives user a good way to integrate third-party tools so you can use the tools you are used to and love.
+
+
+## Strengths
+
+ Azure has a lot of strengths that makes it a good dev-platform to choose to work with over others, i can list a few that comes to mind: 
+
+    1. Cloud-based platform. it makes working with Azure a great experience because it can be accessed anywhere you have an internet connection. Azure DevOps is very scalable meaning you can easy scale it up or down to the projects criteria. 
+
+    2. Integrated solution. Azure DevOps comes with a lot of tools that covers all that you would need from planing, development, testing, distribution ect. its really makes it easier to have all the tools you need in one place that can be accessed from anywhere. 
+
+    3. security. Azure provides several functions and services to ensure that your code and software are safe.
+
+
+## Weaknesses
+
+while i think Azure DevOps is a good platform that provides a lot, i can also come with some downsides, here are a few worth thinking about before choosing: 
```

Comment:
```suggestion
While i think Azure DevOps is a good platform that provides a lot, i can also come with some downsides, here are a few worth thinking about before choosing: 
```

Please check capitalisation using a tool like grammarly or chat GPT. There are lots of issues with capitalisation that could be fixed quite quickly.